### DISCUSSION TITLE
I need the dmg file installer for this version

### DISCUSSION THREAD

**Discussion by mlnfds** [artisan-roaster-scope/artisan#884] [Q&A]
Hi, i use mac os 10.14 (Mojave), i need the artisan v2.4.2 but on this post the file doesnt exist. can anyone help me to send me the dmg file this version please?

**Comment by MAKOMO** [comment]
What is the reason you need to install v2.4.2? The legacy installer of the current v2.6.0, [artisan-mac-2.6.0-legacy.dmg](https://github.com/artisan-roaster-scope/artisan/releases/download/v2.6.0/artisan-mac-2.6.0-legacy.dmg), should work just fine on your setup.

**Reply by MAKOMO** [reply]
As said, the macOS legacy build of v2.6.0 works fine on macOS 10.13 and 10.14.
